easybuild/easyconfigs/m/MRIcron/MRIcron-06_22_15.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "PackedBinary"
+
+name = 'MRIcron'
+version = '06_22_15'
```

Comment:
I would go with a version number that sort better, i.e. `20150622`

I think this corresponds to this version:

```
 MRIcron/NPM/dcm2nii 1JUNE2015	  2015-06-22 06:12
```

In that case, I would actually go with `20150601`, since it's more meaningful?